index.js

```diff
@@ -95,7 +95,7 @@ export default function(opts){
       const namedSlots = this.querySelectorAll('[slot]')
       let slots = {}
       namedSlots.forEach(n=>{
-        slots[n.slot] = this.unwrap(n)
+        slots[n.slot] = n
```

Comment:
This is the primary change (which includes the wrapping `<div slot="inner">` which is typical in normal Svelte output when using named slots).